app/src/main/java/pro/itshark/moneysplitter/presentation/newevent/NewEventActivity.kt

```diff
@@ -0,0 +1,94 @@
+package pro.itshark.moneysplitter.presentation.newevent
+
+import android.arch.lifecycle.Observer
+import android.arch.lifecycle.ViewModelProvider
+import android.arch.lifecycle.ViewModelProviders
+import android.content.Context
+import android.content.Intent
+import android.databinding.DataBindingUtil
+import android.os.Bundle
+import android.support.v7.app.AlertDialog
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import dagger.android.AndroidInjection
+import pro.itshark.moneysplitter.R
+import pro.itshark.moneysplitter.databinding.ActivityAddEventBinding
+import javax.inject.Inject
+
+class NewEventActivity : AppCompatActivity() {
+
+    companion object {
+        fun start(context: Context) = context.startActivity(Intent(context, NewEventActivity::class.java))
```

Comment:
Why do u use `.java`. Could we avoid this?

Reply:
done